content/docs/reference/pass-identity-headers.mdx

```diff
@@ -18,7 +18,7 @@ import TabItem from '@theme/TabItem';
 
 When set to true, the **Pass Identity Headers** setting sends identity headers to all upstream applications.
 
-If a route already has the [route-level Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) setting set, the route setting will take precedence over the global setting.
+If you enable the [route-level Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) setting for a route, the route setting will take precedence over the global setting.
```

Comment:
nit: the word "enable" makes me think this is talking about setting this to `true`, but you might just as well have the global setting `true` and individual routes set to `false`. What do you think about "configure" instead of "enable" here?

```suggestion
If you configure the [route-level Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) setting for a route, the route setting will take precedence over this global setting.
```

Or to phrase this another way entirely, maybe something like this?

```suggestion
To change this behavior for a specific route, use the [route-level Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route).
```